docs/source/daml/intro/7_Composing.rst

```diff
@@ -32,7 +32,7 @@ You can start a new project with a skeleton structure using ``daml new project_n
 
 You can generally set ``name`` and ``version`` freely to describe your project. ``dependencies`` does what the name suggests: It includes dependencies. You should always include ``daml-prim`` and ``daml-stdlib``. The former contains internals of compiler and DAML Runtime, the latter gives access to the DAML Standard Library. ``daml-script`` contains the types and standard library for DAML Script.
 
-You compile a DAML project by running ``daml build`` from the project root directory. This creates a ``dar`` file in ``.daml/dist/dist/project_name-project_version.dar``. A ``dar`` file is DAML's equivalent of a ``JAR`` file in Java: it's the artifact that gets deployed to a ledger to load the package and its dependencies. ``dar`` files are fully self-contained in that they contain all dependencies of the main package. More on all of this in :doc:`8_Dependencies`.
```

Comment:
Why did you remove the statement on DARs being self-contained?

Reply:
Because it was a misunderstanding on my part! Good catch! Reverting this change...